_posts/2016-11-25-CSAPP-Chapter2.md

```diff
@@ -97,25 +130,70 @@ while now the most significant bit comes first is called big endian.
 
 2.2 Integer Representations
 ---------------------------
+Assume we have an integer data type of w bits. We write a bit vector, or as 
+[xw−1xw−2..x1][x_{w-1}x_{w-2}..x_1] to denote its individual bits. When we refer to 
+such a unsigned number, the value of the number is 
+
+B2Ux=∑i=0w−1xi∗2iB2U_x = \sum\_{i =0}^{w-1}{x_i\*2^i} 
```

Comment:
If you want to use `LaTex` syntax, you need to surround the formulation with $$

For example:

$\sum\_{i =0}^{w-1}{x_i\*2^i}$